staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/types.go

```diff
@@ -28,14 +28,17 @@ type CustomResourceDefinitionSpec struct {
 	// Should be always first item in Versions field if provided.
 	// Optional, but at least one of Version or Versions must be set.
 	// Deprecated: Please use `Versions`.
+	// +optional
```

Comment:
I'm not sure the comments on the internal types matter... we don't generate API doc for them as far as I know